src/index.js

```diff
@@ -0,0 +1,8 @@
+var http = require('http');
+var dt = require('./dateTimeModule');
+
+http.createServer(function (req, res) {
```

Comment:
As Lambda's are simple one time executions, the example here should be more simple, like just echo'ing out the date or something like that, per the issue and not standing up a webserver.  Referring to the details in the issue
> basic src/index.js file that can be run with node that dynamically displays the date and time as "Hello! The current data and time is Mon, Oct. 5, 2017 11:03am!"

So in that regard, we should remove express dependency and just keep it 100 dependency free for the time being, and just use something like `console.log` or whatnot